Unresolved Poseidon import

```bash
error[E0432]: unresolved import `poseidon`
 --> /Users/filiplazovic/.cargo/git/checkouts/snark-verifier-972b60abf3a1213b/df03d89/snark-verifier/src/util/hash/poseidon.rs:5:16
  |
5 | use poseidon::{self, SparseMDSMatrix, Spec};
  |                ^^^^ no external crate `poseidon`
  |
help: consider importing this module instead
  |
5 | use poseidon::{crate::util::hash::poseidon;
  |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  ```
  
  I assume this happens because I disabled the default features, which includes poseidon as well.
  ```toml
  snark-verifier = { git = "https://github.com/privacy-scaling-explorations/snark-verifier", default-features = false }
  ```

--- a/snark-verifier/src/util/hash.rs
+++ b/snark-verifier/src/util/hash.rs
@@ -1,7 +1,9 @@
 //! Hash algorithms.
 
+#[cfg(feature = "loader_halo2")]
 mod poseidon;
 
+#[cfg(feature = "loader_halo2")]
 pub use crate::util::hash::poseidon::Poseidon;
 
 #[cfg(feature = "loader_evm")]

